bindings/input_binding.go

```diff
@@ -25,3 +30,12 @@ type InputBinding interface {
 
 // Handler is the handler used to invoke the app handler.
 type Handler func(context.Context, *ReadResponse) ([]byte, error)
+
+func PingInpBinding(inputBinding InputBinding) error {
+	// checks if this input binding has the ping option then executes
+	if inputBindingWithPing, ok := inputBinding.(health.Pinger); ok {
+		return inputBindingWithPing.Ping()
+	} else {
+		return fmt.Errorf("Ping is not implemented by this Input Binding")
```

Comment:
```suggestion
		return fmt.Errorf("Ping is not implemented by this input binding")
```

this should be lowercase for consistency and it currently breaks tests

Reply:
Done. To understand that what tests are failing and how these rules are setup, I tried to search, are these the Conformance tests that are failing? I could not find this in UTs.